lib/matplotlib/mlab.py

```diff
@@ -157,6 +159,7 @@ def detrend_mean(x, axis=None):
     return x - x.mean(axis, keepdims=True)
 
 
+@_api.deprecated("3.6", alternative="scipy.signal.detrend")
```

Comment:
That doesn't seem to be right alternative?

Reply:
fixed!